staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -169,17 +174,60 @@ func GetValueFromIntOrPercent(intOrPercent *IntOrString, total int, roundUp bool
 	return value, nil
 }
 
+//Deprecated: GetValueFromIntOrPercent was deprecated in favor of
```

Comment:
You'll break go lint if this comment doesn't start with the function name, so move the deprecation notice to be at the end of the comment, I guess.